packages/datatrak-web/src/views/SurveyPage.tsx

```diff
@@ -36,7 +36,7 @@ const SurveyScreenContainer = styled.div<{
   overflow: ${({ $scrollable }) => ($scrollable ? 'auto' : 'hidden')};
   align-items: flex-start;
   height: calc(100vh - ${HEADER_HEIGHT} - ${SURVEY_TOOLBAR_HEIGHT});
-  width: 100vw;
+  width: 100%;
```

Comment:
@acdunham Looks like you added the `100vw` last week to fix some layout things on mobile. Is it possible this change will have broken something else? I’m not sure if `SurveyScreenContainer` ever wraps something _other_ than `SurveySuccessScreen`.

The padding on this container was causing the overflow in narrow windows.

Reply:
It's possible - you might need to play around with the different views to make sure it all looks legit at all screen sizes 🙏

Reply:
Had a squiz (in particular `SurveyLayout`) and looks like the `%` actually does fix the same issue for the different views

Reply:
I think the 100vw was there to fix this issue, as that layout is shared between multiple screens. Ideally there is a fix for both issues
![Screenshot 2023-11-13 at 3 07 56 pm](https://github.com/beyondessential/tupaia/assets/129009580/b4728c56-c86f-424c-ac9f-32b38e418c6b)

Reply:
Huh, I think I’ve accidentally made a mess of this branch. The behaviour from your screenshot doesn’t happen on the epic branch.

I think what’s happened is:

1. I originally branched of `dev` (incorrectly) and pushed that commit
2. I deleted my local branch, and created a new one, correctly branching off `datatrak-web-epic`
3. …but I gave the branch the same name, so when I pushed to GitHub it pushed to the existing remote branch

Which would also explain this 😅

![Screenshot 2023-11-13T162957](https://github.com/beyondessential/tupaia/assets/33956381/43bce7a0-2098-4d3f-8478-308f8aa4f00d)

Given the small scope of this change, I think tidiest for me to redo this on (yet) another branch **with a different name**

Reply:
Looks like the clipping of the bottom toolbar (with Back, Cancel and Next) happens with both `100%` or `100vw`. I’ve just merged the latest code from the epic branch, and this is what it’s doing at 320 and 220px viewports:

![Screenshot 2023-11-13T165817](https://github.com/beyondessential/tupaia/assets/33956381/80833d72-d7ec-4aa7-a24b-8ddc0f4a3cc2)

![Screenshot 2023-11-13T165834](https://github.com/beyondessential/tupaia/assets/33956381/14613b46-1b7f-4079-a10b-506b9a481978)

The most straightforward workaround I can think of would be to use a single-column button group in the small size class, but it doesn’t quite jive with the current styling of these three buttons:

<img width="471" alt="Frame 5" src="https://github.com/beyondessential/tupaia/assets/33956381/37c1b3ba-e0f3-47a3-a807-5294d8d315d5">

Versus:

<img width="471" alt="Frame 6" src="https://github.com/beyondessential/tupaia/assets/33956381/de6cc5fa-5c0a-452e-a111-2e2f14fe9bfd">

…but this would mean also changing the non-narrow styling.

Reply:
Probably don't need to worry about 220px width - I don't know if there are any devices that narrow, and if there are it's not likely they will be commonly used. 320px wide is usually the limit people work with